src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -102,6 +103,46 @@ public function editAction(Request $request, int $combinationId): Response
         ]);
     }
 
+    /**
+     * @AdminSecurity("is_granted('update', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     * @param int $combinationId
+     *
+     * @return JsonResponse
+     */
+    public function bulkEditAction(Request $request, int $combinationId): JsonResponse
+    {
+        try {
+            $bulkCombinationForm = $this->getBulkCombinationFormBuilder()->getFormFor($combinationId);
+        } catch (CombinationNotFoundException $e) {
+            return $this->returnErrorJsonResponse(
+                [$this->getErrorMessageForException($e, $this->getErrorMessages($e))],
+                Response::HTTP_NOT_FOUND
+            );
+        }
+
+        try {
+            $bulkCombinationForm->submit(
+                $request->request->get($bulkCombinationForm->getName(), []),
+                // Only submits values that was provided (avoids filling missing values as NULL, which would break constraints)
+                false
+            );
```

Comment:
ping @matks @jolelievre is such thing acceptable? I couldnt find other way to make it work.
Using PATCH to submit form via ajax doesn't work (no data in request). 
So I submit this form manually here using second param `false` to avoid setting missing data as null. (because if data is set, then constraints like NotBlank are failing).

Reply:
nvm PATCH works, there is data in "content" not in "request->request" and indeed form handles correctly when using PATCH, probably it wasn't working due to data provider and I mistakenly crossed out the request method instead